src/ui/components/message/CastMessage.tsx

```diff
@@ -35,13 +35,9 @@ export class CastMessage extends PureComponent<unknown, CastMessageState> {
 
   private _processCastState = (state: CastState, target: 'Chromecast' | 'Airplay') => {
     let message: string | undefined;
-    switch (state) {
-      case 'connecting':
-        message = `Connecting to ${target} ...`;
```

Comment:
I probably would have kept the `switch..case` and have two separate localization keys for "connecting" and "connected".